docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,22 @@
 # Staging release notes
 
+## October 2023
+
+**11 October 2023**
+
+Frontend version: 1.0.0-20231009.1523 | Backend version: 1.69.0-20231012.1057
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | Check existing users on Automation API | With the launch of CAM on 23 October 2023, the Automation API will now monitor terminated user events. 
+This API facilitates a one-off event, enabling tenants to clean up their local user store by removing accounts that no longer exist in TechPass AAD. This step is vital before integration with TechPass for CAM compliance. After integration, tenants will receive new user terminated webhook events.|
+| **Enhancement** | Verify user existence in WoG AAD for invite public officer API  | We have enhanced our Invite Public Officer API by verifying user existence in WoG AAD when the provided email belongs to a WoG email domain. TechPass account will not be created if user does not exist in WoG AAD. However to prevent User Enumeration attack, TechPass portal's Self Sign-Up page will always return success, while the Automation API returns a generic error. <br><br>**Note**: For MoE AAD users, TechPass account will always be created as we are not able to verify user existence in MoE AAD. |
+| **Enhancement** | New tenant flag - acknowledgeComplianceFlag  | Tenants now have the option to indicate whether they maintain their own user directory. Tenants managing their directories must subscribe to our `user-updated` and `user-deleted` webhooks. |
+| **Enhancement** | Product owners are now able to specify "from" field for email broadcasts | Previously, all emails were dispatched from the enquiries TechPass email address. Now, product owners can use their custom email addresses for broadcasts. Refer to [Email template](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/email-template) for more details. |
```

Comment:
We don't intend to publish this information because the feature is not ready yet. There are certain domains (i.e. hive.gov.sg, tech.gov.sg) that has DKIM and SPF configured so we can't send on their behalf.

Yong Zhi is still looking into this so we will not broadcast yet.

Reply:
ah ok!